provisioner/terraform/executor.go

```diff
@@ -386,6 +386,8 @@ func (e *executor) plan(ctx, killCtx context.Context, env, vars []string, logr l
 		}
 	}
 
+	// DoneOut must be completed before we aggregate timings to ensure all logs have been processed.
+	<-doneOut
```

Comment:
Should we call `outWriter.Close()` before doing this, to avoid a deadlock in case the implementation of `execWriteOutput()` is ever changed to not call it?

Reply:
Good catch!

Reply:
I just moved the timings to outside the exec functions actually. This is cleaner